etl/multidim.py

```diff
@@ -162,3 +167,74 @@ def fetch_variables_from_table(table: str, engine: Engine) -> pd.DataFrame:
     df_dims = pd.DataFrame(dims, index=df.index)
 
     return df.join(df_dims)
+
+
+def generate_views_for_dimensions(
```

Comment:
Nice! It'd be good to unify `generate_views_for_dimensions` and `expand_views` into a single function in the future. They both do pretty much the same thing - take a list of (flattened) indicator paths (be it from ETL or DB) and create views from it. `expand_views` has more options for aggregations and `generate_views_for_dimensions` better logging, but they are essentially the same.

Reply:
Yes, I'd be in favour of harmonizing / consolidating functions

Reply:
Yes, totally! I initially thought of harmonizing both functions. But then I thought they are quite different in what they do. If we are moving towards mdim steps that do not read from DB, maybe there's not much gain in adapting `expand_views`. So, until we decide on that, we can keep these two versions.